### PR TITLE
lcms2 2.16: with  libtiff 4.5.1

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Destination channel:** main

### Links

- [PKG-6505](https://anaconda.atlassian.net/browse/PKG-6505) 
- [Upstream repository](https://github.com/mm2/Little-CMS/tree/lcms2.16)
- Requirements:
  - https://github.com/mm2/Little-CMS/blob/lcms2.16/doc/LittleCMS2.16%20API.pdf

### Explanation of changes:

- Pin libtiff to 4.5.1
- Use vs2019 on win
- Add --disable-static on unix
- Add more test commands
- Add dev& doc urls
- Update license name
- Add description

### Notes:

- https://github.com/AnacondaRecipes/libtiff-feedstock/pull/20

[PKG-6505]: https://anaconda.atlassian.net/browse/PKG-6505?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by SC426 — APPROVED** [review]

**Review by ELundby45 — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,53 +1,64 @@
...
   build:
     - {{ compiler('c') }}
-    - make  # [not win]
+    - make     # [not win]
     - libtool  # [unix]
   host:
-    - jpeg
-    - libtiff
+    - jpeg {{ jpeg }}
+    - libtiff 4.5.1
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - libtiff {{ libtiff }}
```

**Review by cbouss — APPROVED** [review]
nitpick, but there is no reason to pin libtiff to 4.5.1 specifically.

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: The latest version **4.5.1** (with [lerc 4](https://github.com/AnacondaRecipes/lerc-feedstock/pull/3) and [libdeflate 1.22](https://github.com/AnacondaRecipes/libdeflate-feedstock/pull/6)) is required in host  as there is a fix for Windows:
`LINK : fatal error LNK1181: cannot open input file 'libtiff.lib' [%SRC_DIR%\Projects\VC2019\tifficc\tifficc.vcxproj]`
see https://github.com/AnacondaRecipes/libtiff-feedstock/pull/20/commits/822fcae16c9b6219a5425e312949e6a35e7c4c4f
I've checked that it works fine with `libtiff 4.5.1` but fails with **4.5.0** on `win-64`.

Comment (skupr-anaconda) [reply recipe/meta.yaml]: I rewrote to:
```
    # libtiff 4.5.1 fixes a missing libtiff.lib on Windows
    - libtiff 4.5.1          # [win]
    - libtiff {{ libtiff }}  # [not win]
```

**Review by scotthain — APPROVED** [review]
LGTM
🚀 🍰

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: I think it has a cascading effect on all libtiff's downstreams on win-64: lcms2, poppler, pillow, gdal, imagecodecs

Comment (cbouss) [reply recipe/meta.yaml]: Can you open a PR on the cbc to change libtiff pin to 4.5 instead?

Comment (skupr-anaconda) [reply recipe/meta.yaml]: Yes, I'll do

Comment (skupr-anaconda) [reply recipe/meta.yaml]: Here we go https://github.com/AnacondaRecipes/aggregate/pull/324

**Review by cbouss — APPROVED** [review]

**Comment by cbouss** [comment]
Coming back to this, I think it's worth bringing this upstream fix: 
https://github.com/mm2/Little-CMS/issues/420
https://github.com/mm2/Little-CMS/commit/e88056ef81e42cd1fe06945da9d44ac95c80fa48.patch

**Pull request merged**